pkg/virt-handler/isolation/isolation.go

```diff
@@ -120,6 +128,59 @@ func (s *socketBasedIsolationDetector) Detect(vm *v1.VirtualMachineInstance) (*I
 	return NewIsolationResult(pid, slice, controller), nil
 }
 
+// standard golang libraries don't provide API to set runtime limits
+// for other processes, so we have to directly call to kernel
+func prLimit(pid int, limit uintptr, rlimit *unix.Rlimit) error {
+	_, _, errno := unix.RawSyscall6(unix.SYS_PRLIMIT64,
```

Comment:
in previous commit you dropped RawSyscall, can we do it also here?

Reply:
no because this syscall is not implemented in standard libraries.

I dropped raw syscall in a separate commit because it was brought to my attention that we have similar code in another place, but instead of reusing the new prLimit function there, it was easier and better to switch to a proper library call.